### ISSUE TITLE
Toc Numbering different to Document number

### ISSUE THREAD

**Issue by JeremyOWO** [splitbrain/dokuwiki-plugin-dw2pdf#393]
Hello all,

Is it possible to have just one global numbering for the document ?

Because today we have two parts:
- The first numbering is for cover page and toc ( In my case : I/II and and II/II )
- The second numbering start after TOC. ( 1/14 to 14/14 )

But I would like to have direclty 1/16 to 16/16.
Do you have a solution to merge this two numbering into just once ?

Thanks

**Comment by Klap-in** [comment]
I have no idea. I'm surprised it is using two numberings. Please provide an example for testing.

**Comment by Klap-in** [comment]
Which settings do you use for dw2pdf in the Configuration Manager, especially for ToC?

**Comment by Klap-in** [comment]
ah, I see now what you mean.
This was due to a limitation of the library mPDF we use for generating pdfs. It first creates all the pages of the file which are added one-by-one, and on the end after processing the pages, it includes the ToC on the start. The final page numbering is only known after finishing the document. Because the library had troubes with guessing the right length of the ToC upfront, two set of pagenumbers are used. 

I see in the code italic number is selected: https://github.com/splitbrain/dokuwiki-plugin-dw2pdf/blob/6d1b3a64fc56857a4bb99f6e64bad6f5f9eafca9/action.php#L443

I just saw that the mPDF library put effort in the ToC creation in v7. Might be there are improvements that could benefit us as well. Unfortunately I have no time for looking into this. Contributions are welcome.
I expect removing the italic page numbering and not resetting it in the TOCpagebreakByArray() might do already something. But it needs testing.
